src/components/Form/Form.vue

```diff
@@ -0,0 +1,33 @@
+<template>
+  <form class="">
+    <select class="" name="">
+      <option>Choose a State</option>
+      <option
+        v-for="(state, index) in states"
+        :key="index"
+        :value="state.state"
+      >{{state.state}}</option>
+    </select>
+    <select class="" name="">
+      <option>Choose a City</option>
+      <option
+        v-for="(city, index) in cities"
+        :key="index"
+        :value="city.city"
+      >{{city.city}}</option>
+    </select>
+    <input type="submit" name="submit" value="Submit">
+  </form>
+</template>
+
+<script>
+export default {
+  name: 'Form',
+  props: {
```

Comment:
I'm still a little unclear on why it's necessary to declare the data types in props (and why sometimes it works without). Let's look into this deeper later!